python/_eos.cc

```diff
@@ -809,5 +803,5 @@ BOOST_PYTHON_MODULE(_eos)
     // }}}
 
     // EOS version
-    def("version", impl::version);
+    scope().attr("__version__") = PACKAGE_VERSION;
```

Comment:
This seems to trigger an error `module 'eos' has no attribute 'version'` for me

Reply:
Fixed. The documentation generators and the `eos.data` module use `eos.version()`.